### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by Ahmed-Alfaouq**
What we did:
- Added modal JS logic
- Added modal style
- Fix linters errors

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Ahmed-Alfaouq, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -56,3 +57,103 @@ contactLink[0].addEventListener('click', () => {
...
+  ],
+  text: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+  when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+  It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent`,
+  image: {
+    src: './images/modal.png',
+    alt: 'modal image',
+  },
+};
```
Comment (mke2111) [inline index.js:77]: - [ ] Add to your object a key-value property for the link to the live version and the link to the project source code.

Comment (Ahmed-Alfaouq) [reply index.js]: @mke2111
Kindly, can you check it now?
thanks

Comment (mke2111) [reply index.js]: Submit another CRR when you are ready.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Omar, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 :shipit: .
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**